zerver/lib/realm_icon.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+from django.conf import settings
+
+if False:
+    from zerver.models import Realm
+
+from typing import Text
+
+from zerver.lib.avatar_hash import gravatar_hash, user_avatar_hash
+from zerver.lib.upload import upload_backend
+
+def realm_icon_url(realm):
+    # type: (Realm) -> Text
+    return get_realm_icon_url(
+        realm.icon_source,
+        realm.domain)
```

Comment:
Should the version be an input into these URLs?  It seems like that would be a nice system.